cypress/integration/settings/circulation/general/edit-circulation-rules.spec.js

```diff
@@ -0,0 +1,93 @@
+import TestType from '../../../../support/dictionary/testTypes';
+import SettingsMenu from '../../../../support/fragments/settingsMenu';
+import CirculationRules from '../../../../support/fragments/circulation/circulation-rules';
+import OverdueFinePolicy, {
+  defaultOverdueFinePolicy,
+} from '../../../../support/fragments/circulation/overdue-fine-policy';
+import LostItemFeePolicy, {
+  defaultLostItemFeePolicy,
+} from '../../../../support/fragments/circulation/lost-item-fee-policy';
+import RequestPolicy, {
+  defaultRequestPolicy,
+} from '../../../../support/fragments/circulation/request-policy';
+import LoanPolicy, {
+  defaultLoanPolicy,
+} from '../../../../support/fragments/circulation/loan-policy';
+import NoticePolicy, {
+  defaultNoticePolicy,
+} from '../../../../support/fragments/circulation/notice-policy';
+import MaterialTypes, {
+  defaultMaterialType,
+} from '../../../../support/fragments/inventory/materialType/materialTypesSettings';
+import permissions from '../../../../support/dictionary/permissions';
+
+describe('ui-circulation-settings: Edit circulation rules', () => {
+  let originalCirculationRules;
+  let newUserId;
+
+  beforeEach(() => {
+    cy.createTempUser([
+      permissions.uiCirculationViewCreateEditDelete.gui,
+    ]).then(({
+      username,
+      password,
+      userId,
+    }) => {
+      newUserId = userId;
+
+      cy.getAdminToken();
+
+      CirculationRules.getCirculationRulesApi().then((circulationRules) => {
+        originalCirculationRules = circulationRules;
+      });
+      MaterialTypes.createMaterialTypeApi();
+      NoticePolicy.createNoticePolicyApi();
+      LoanPolicy.createLoanPolicyApi();
+      RequestPolicy.createRequestPolicyApi();
+      LostItemFeePolicy.createLostItemFeePolicyApi();
+      OverdueFinePolicy.createOverdueFinePolicyApi();
```

Comment:
fix naming. For example - OverdueFinePolicy.createOverdueFinePolicyApi(); -> OverdueFinePolicy.createApi();